### ISSUE TITLE
Number of students

### ISSUE THREAD

**Issue by Mhdez221993** [Mhdez221993/Capstone-project-Kanban-board#41]
In this project
https://github.com/Mhdez221993/Capstone-project-Kanban-board/projects/1
we're three students
Hiro
Abraham
Moises

**Comment by Davidosky007** [comment]
Hi @Mhdez221993, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the wrong link, The link provided lead to an empty repository, please ensure you provide the correct link and submit it for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by vichuge** [comment]
# Project invalidated 🚫 

Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

![cat_massage](https://user-images.githubusercontent.com/20563252/121368971-16d2dc00-c901-11eb-9123-7f578bf64118.gif)

## Why does this happen ❓ ❓ ❓ 

You have submitted a project, that shares the Issue link in place of the project link, please, for next time try to share the link where is the [project itself](https://github.com/Mhdez221993/Capstone-project-Kanban-board/projects/1).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by nidalaa** [comment]
Hello @Mhdez221993 ,

Unfortunately, @vichuge made a mistake in this review :( 

Your request was valid and should have been checked. We are really sorry for the misunderstanding.

I have made a quick review in order to unblock you:)

**Please request another review, once you make the requested changes.**

--------------

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️

Your board looks awesome and I can see that you already started to move cards but there is one detail missing!

You were supposed to **assign** your members to cards not to indicate who is doing what in the card's title.
This serves the same purpose so it looks like not a big deal but:

- the standard way is to use the **assign** feature of GH for this
- thanks to that it will be easier to filter the cards and the issues if needed.

Here is a screenshot to show you what I mean by **assign**:

<img width="391" alt="Screenshot 2021-08-02 at 14 07 11" src="https://user-images.githubusercontent.com/7466903/127866973-980f0bc4-bea8-4083-86ca-2a971f65fc90.png">



Please continue working on the code but also make sure that you **assign** GH users to the cards.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------

**Comment by Davidosky007** [comment]
# Hi @Mhdez221993 @Metsanpeitto 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
